tests/env.py

```diff
@@ -10,10 +11,15 @@
 class Environment(object):
 
     def setup(self):
+        self.setup_collations()
         for i in range(0, 2):
             self.setup_translations(i)
         self.setup_x()
 
+    def setup_collations(self):
+        cursor = connection.cursor()
+        cursor.execute("CREATE COLLATION tr_tr (LOCALE = 'tr_TR.utf8');")
```

Comment:
Can you add a short comment either explaining why we need this and/or adding a link to the bug?